requirements.txt

```diff
@@ -1,11 +1,6 @@
-aniso8601==8.0.0
-click==7.1.2
 Flask==1.1.2
 Flask-Cors==3.0.8
 Flask-RESTful==0.3.8
-itsdangerous==1.1.0
-Jinja2==2.11.2
-MarkupSafe==1.1.1
+aniso8601==8.0.0
 pytz==2020.1
 six==1.15.0
-Werkzeug==1.0.1
```

Comment:
I have a belief that Wekzeug is a required dependency for flask. Is that correct?